server/store/project.go

```diff
@@ -0,0 +1,58 @@
+package store
+
+import (
+	"fmt"
+	"strings"
+)
+
+// ProjectIdentifier is a structure to differentiate projects from different VCS
+type ProjectIdentifier struct {
+	VCSType string // Currently supported : "gh" and "bb"
+	Org     string
+	Project string
+}
+
+// ToSlug returns the slug in format "(gh|bb)/org-name/project-name)"
+func (pi *ProjectIdentifier) ToSlug() string {
```

Comment:
nitpick: use single character reciever variable names
```suggestion
func (p *ProjectIdentifier) ToSlug() string {
```

Reply:
Here, it was to make it clear that it's not the `p*` of `Plugin` as in the rest of the code. Do you think this distinction is useful? (I'm really nitpicking here :smile:)

Reply:
I'm also nitpicking here and will leave it up to you. This file is in the store package, so it does stand alone, but I also see your point.